src/taco-cli/test/create.ts

```diff
@@ -58,7 +58,7 @@ interface IKeyValuePair<T> {
 
 describe("taco create", function (): void {
     // Test constants
-    var createTimeout: number = 60000;
+    var createTimeout: number = 180000;
```

Comment:
Yes; I was still seeing occasional failures with 120000 seconds, so I bumped it up further. I think the main cause of timeouts is network instability at the moment.